build-wheelhouse/action.yml

```
@@ -132,7 +132,7 @@ runs:
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

```suggestion
      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
      with:
        persist-credentials: false
```
Following your previous changes (which make sense), shouldn't we also be using `persist-credentials: false` for every composite actions where we use `actions/checkout` and don't perform extra git calls ?
If yes, you should track the other locations and apply the changes :)

---

The permission should be set in the workflow calling the composite action. This can still be at the workflow/job level, providing enough granularity for control. This was why I made changes in `./.github/workflows/*.yml` but not `./*/action.yml`. This is also the reason why `zizmor` doesn't flag permissions audit within composite actions.

---

Gotcha !